webapp/src/components/QuestionCard.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import {
+  TextField,
+  FormControlLabel,
+  Radio,
+  Card,
+  CardContent,
+  Typography,
+  RadioGroup,
+  FormControl,
+  MenuItem,
+} from '@mui/material';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+
+import { Question } from '../assets/data';
+
+interface QuestionCardProps {
+  question: Question;
+  handleNewAnswer: (
+    question: Question,
+    chosenAnswerId?: number,
+    responseText?: string
+  ) => void;
+  currentStatus: string;
+}
+
+const QuestionCard = ({
+  question,
+  handleNewAnswer,
+  currentStatus,
+}: QuestionCardProps) => {
+  const [radioValue, setRadioValue] = React.useState('');
```

Comment:
We should be passing in the corresponding AssessmentResponse here and set it as the default value of your radios and text boxes rather than an empty string.